doc/modules/decomposition.rst

```diff
@@ -726,3 +726,53 @@ the data.
       matrix factorization"
       <http://scgroup.hpclab.ceid.upatras.gr/faculty/stratis/Papers/HPCLAB020107.pdf>`_
       C. Boutsidis, E. Gallopoulos, 2008
+
+
+.. _LatentDirichletAllocation:
+
+Latent Dirichlet Allocation (LDA)
+=================================
+
+Latent Dirichlet Allocation is a generative probabilistic model for collections of discrete
+dataset such as text corpora. It is also a topic model that used for discovering abstract
+topics from a collection of docuements.
+
+:class:`LatentDirichletAllocation` implements online variational Bayes algorithm and supports
+both online and batch update method.
+In batch method, global variational variables are udpated after each full pass through the data.
+Unlike batch method, online method computes intermediate variational variables from mini-batch
+data points and updates the variational variables as a weighted average of the intermediate
+variables and their current values.
+Therefore, online method usually converges faster than batch method.
```

Comment:
I think that you are going too much into details here: what the end user really cares about is that it is fast, but not really about the details of why.